app/controllers/SubmitFormController.scala

```diff
@@ -136,12 +136,12 @@ class SubmitFormController @Inject()(mcc: MessagesControllerComponents,
                 )).addingToSession(SessionKeys.viewModel -> Json.toJson(viewModel).toString())
 
               } else {
-                logger.warn("[SubmitFormController][renderViewWithoutSessionData] " +
+                errorLog("[SubmitFormController][renderViewWithoutSessionData] " +
                   s"Obligation end date for period $periodKey has not yet passed.")
                 errorHandler.showBadRequestError
               }
             case _ =>
-              logger.warn(s"[SubmitFormController][Show]: Obligation not found for period key $periodKey")
```

Comment:
same here

Reply:
i'll change this back to a warnLog, the errorLog should go to the next log